lib/galaxy/webapps/galaxy/api/dataset_collections.py

```diff
@@ -67,6 +91,68 @@ def create(
     ) -> HDCADetailed:
         return self.service.create(trans, payload)
 
+    @router.get(
```

Comment:
If you're creating this workbook here, why not make it a POST ?
I feel like we can provide a much nicer entry point for API consumers if this had a schema that wasn't just a string